src/main/java/examples/VertxKafkaClientExamples.java

```diff
@@ -73,9 +162,9 @@ public void example1(Vertx vertx) {
    */
   public void example2(KafkaConsumer<String, String> consumer) {
```

Comment:
Yes I like it :-) A lot of times I needed to shift example index in order to add a new one in the middle